concepts/observability-configurable-logging/08-revised-api.md

```diff
@@ -0,0 +1,290 @@
+-> Add dedotting support as a feature to the `LogPipeline`
+
+### Meaningfulness of LogPreset
+
+It turned out that an OpenSearch integration via a HTTP plugin required for an internal SAP scenario can be much simpler then expected. A real templating is not needed and probably a plain simple pipeline config will be better suited. Also, what presets to provide, we cannot think of other scenarios which can be delivered. The binding to a secret can be done already in the pipeline itself, workload selectors could be added as well
+-> do we need the LogPreset concept?
+
+### Support of additional inputs
+It turned out that there are typical inputs available which you would like to process as well, mainly the kubelet logs
+-> add a source selector
+
+## Revised API
+
+More and more oppinion/abstraction was added to the `LogPipeline` concept and the meaningfullnes of the `LogPreset` is in the room. We could now try to push down the `LogPipeline` to be more native and accepting that the user can do really stupid things in combination of supported plugins and revise the LogPreset concept.
+Or we push up even more the `LogPipeline` API by introducing the convinience of the LogPreset while keeping native support as a customization option clearly indicating an unsupported scenario.
+This proposal favors the second idea by dropping the LogPreset fully and having only one layer which is already an abstraction. Giving no chance to configure fluent-bit natively. For that you can simply go with a custom installation.
+
+```YAML
+kind: LogPipeline
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: OpenSearchHTTP-App
+spec:
+  input:
+    application: # maps to central tail pipeline
+      namespaces: [] # maps to rewrite_tag rule
+      excludeNamespaces: []
+      pods: []
+      excludePods: []
+      containers: []
+      excludeContainers: []
+      matchLabels:
+        app: icke
+    system: {} #maps to systemd based input like kubelet logs
+    custom: | # entering unsupported mode
+      Name dummy
+      Dummy {"message":"dummy"}
+  filters:
+    - multilineParser:
+        type: "java"
+    - parser:
+        type: "json" # support of built-in parsers
+    - modify:
+        add:
+          key: cluster_identifier
+          value: ${KUBERNETES_SERVICE_HOST}
+    - modify:
+        remove:
+          key: cluster_identifier
+    - select:
+        include:
+          key:
+          regexp:
+        exclude:
+          key:
+          regexp:
+    - custom: | # no "Match" available, entering unsupported mode
+        Name    record_modifier
+        Record  cluster_identifier ${KUBERNETES_SERVICE_HOST}
+
+  output: #only one output
+    http:
+      Dedot: true
```

Comment:
```suggestion
      Dedot: true
      Match: Logpipeline.*
```

Reply:
The idea was to unburden the user from understand tagging in fluent-bit at all. So you are limited to the logs of the pipeline selected in the inout section, that's it. What is your motivation of having match configurable? To support applying it to a subset of logs only? We dropped that idea, elements  will be applied to all in the pipeline always. We also suggest to have only one output in a pipeline (it is singular now), so applying an output to a subset makes no sense anyway then as you can just limit the logs already in the input section.